list_updatable_packages

```diff
@@ -4,14 +4,18 @@ import json
 import os
 import subprocess
 import sys
+import shutil
 from pathlib import Path
 from typing import Generator, Iterable, Mapping, Optional
 
 import requests
+import tempfile
 
 SUPPORTED_TEMPLATES = ('foreman_plugin', 'hammer_plugin', 'smart_proxy_plugin')
 SESSION = requests.Session()
-
+FOREMAN_GEMFILE_LOCK_URL = 'https://ci.theforeman.org/job/foreman-develop-source-release/lastSuccessfulBuild/artifact/Gemfile.lock'
```

Comment:
When I think about the future steps, I'd consider implementing `get_ci_artifact_url(job_name: str, artifact: str, build: str='lastSuccessfulBuild)` but I'd happy to do the same in a future PR.